src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -342,6 +344,29 @@ class GuiPluginList extends React.PureComponent<Props, State> {
     }
   }
 
+  handleError = (error: any): void => {
```

Comment:
Please change `error` to `unknown`.

Of course, then you can't do things like `error[0].quoteError.errorType`, so you'll want to make some cleaners to identify error type instead:

```ts
const asMaybeRegioinRestricted = asMaybe(asArray(asObject({
  quoteError: asObject({ errorType: asValue('regionRestricted') })
})))
```